examples/CsvExample.php

```diff
@@ -0,0 +1,218 @@
+<?php
+
+require_once __DIR__ . '/../vendor/autoload.php';
+
+use FINDOLOGIC\Export\Data\Item;
+use FINDOLOGIC\Export\Exporter;
+use FINDOLOGIC\Export\Data\Ordernumber;
+use FINDOLOGIC\Export\Data\Image;
+use FINDOLOGIC\Export\Data\Attribute;
+use FINDOLOGIC\Export\Data\Keyword;
+use FINDOLOGIC\Export\Data\Usergroup;
+use FINDOLOGIC\Export\Data\Property;
+
+/**
+ * This example class builds a CSV export inspired the FINDOLOGIC documentation, which can be found at
+ * https://docs.findologic.com/doku.php?id=export_patterns:csv .
+ */
+class CsvExample
+{
+    public function createExport()
+    {
+        $exporter = Exporter::create(Exporter::TYPE_CSV);
+
+        $itemsToExport = [];
+
+        // Here you could have an array with the item data to iterate threw
+        $itemsData = ['item1'];
+
+        foreach ($itemsData as $itemData) {
+            $item = $exporter->createItem('01120c948ad41a2284ad9f0402fbc7d');
+
+            $this->addOrdernumbers($item, $itemData);
+            $this->addNames($item, $itemData);
+            $this->addSummaries($item, $itemData);
+            $this->addDescriptions($item, $itemData);
+            $this->addPrices($item, $itemData);
+            $this->addUrls($item, $itemData);
+            $this->addKeywords($item, $itemData);
+            $this->addBonuses($item, $itemData);
+            $this->addSalesFrequencies($item, $itemData);
+            $this->addDateAddeds($item, $itemData);
+            $this->addSorts($item, $itemData);
+            $this->addUsergroups($item, $itemData);
+            $this->addImages($item, $itemData);
+            $this->addAttributes($item, $itemData);
+            $this->addProperties($item, $itemData);
```

Comment:
The method doesn't add properties yet, as the exporter isn't initialized with the properties.

Reply:
:heavy_check_mark: